example/tests/test_backends.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from rest_framework.test import APITestCase
+
+from ..models import Blog, Entry
+
+ENTRIES = "/nopage-entries"
+
+
+class DJATestParameters(APITestCase):
+    """
+    tests of JSON:API backends
+    """
+    fixtures = ('blogentry',)
+
+    def setUp(self):
+        self.entries = Entry.objects.all()
+        self.blogs = Blog.objects.all()
+
+    def test_sort(self):
+        """
+        test sort
+        """
+        response = self.client.get(ENTRIES + '?sort=headline')
+        self.assertEqual(response.status_code, 200,
+                         msg=response.content.decode("utf-8"))
+        j = json.loads(response.content.decode("utf-8"))
```

Comment:
to get json result from response you should be able to simply do `result = response.json()` I would also prefer not to use single letter variables as python is not strongly typed so the name should tell what it which makes the code more readable (e.g. in this example result instead j or similar)

The same counts to the tests below.

Reply:
I've seen cases where `response.json()` doesn't work. Taking a look if this is one of them.

Reply:
but this is not one of them.